src/tzdata/build.jl

```diff
@@ -38,25 +38,53 @@ function build(
         end
     end
 
-    archive = joinpath(archive_dir, "tzdata$version.tar.gz")
-
-    # Avoid downloading a tzdata archive if we already have a local copy
-    if version == "latest" || !isfile(archive)
-        @info "Downloading $version tzdata"
-        archive = tzdata_download(version, archive_dir)
-
+    if VERSION >= v"1.4"
+        now_utc = now(Dates.UTC)
+        # todo: I don't have latest here and it should not be used as latest
         if version == "latest"
-            m = match(TZDATA_VERSION_REGEX, basename(archive))
+            m = match(TZDATA_VERSION_REGEX, "tzdata$version.tar.gz")
             if m !== nothing
                 version = m.match
                 @info "Latest tzdata is $version"
             end
+
+            artifact_dir = @artifact_str "tzdata_$version"
```

Comment:
If `version == "latest" then `match` will return `nothing` and this will resolve to "tzdata_latest" which doesn't exist as an artifact.

Reply:
You're right, I missed that. I guess in that case it should download the newest one, right?

Reply:
So I'm directly downloading the newest one if it's still latest.